### PR TITLE
Feature/iteration bullets

### PR DESCRIPTION
**Pull request by Fakhatova**
### 🚨🤦🏽🚨 *Problem:*
*What is the purpose of this PR? Bug? Refactor? Feature? Please describe the problem you're trying to solve*

- Just worked on user repo file to be able to go over some test and check them on bugs and refactor some of them 

### 🌟💡🌟 *Solution:*
*How are you solving this simply and elegantly?*

- TDD and console.log

### 🤔💬🤔 *Test/Review Plan:*
*What's your proof this works? Unit tests? Staging? If you want reviewer to click-test, include specific instructions*

- Unit Test 

### 👁‍👁‍ *Changes:*
*What was the upfront planning that went into this?*
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)  

###  💠✅💠 *CheckList:* 
*Overview of your task*
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/User-repo.js [hunk 1]
```diff
@@ -1,30 +1,51 @@
...
+    // console.log('all users', this.users);
   }
   getDataFromID(id) {
+
+    // return this.users.find((user) => id === user.id);
     return this.users.find((user) => id === user.id);
   }
+
+  // helper function being used in makeSortedUserArray 👇
   getDataFromUserID(id, dataSet) {
```
Comment (nataliemcintyre2021) [inline src/User-repo.js:13]: Continue to see if this method is necessary/where it is used

File: src/Activity.js [hunk 2]
```diff
@@ -92,3 +92,34 @@ class Activity {
 
 
 export default Activity;
+
+
+
+
+
+/* Iteration 5 - Activity
```
Comment (ninabrissey) [inline src/Activity.js:100]: Great Idea adding this here for ease of use.

File: src/User-repo.js [hunk 3]
```diff
@@ -1,30 +1,51 @@
...
+    const totalStepGoal = this.users.reduce((total, user) => {
+      total += user.dailyStepGoal;
+      return total
     }, 0);
+    console.log(totalStepGoal / this.users.length, "nina")
     return totalStepGoal / this.users.length;
   }
+
+  // Natalie working on
   makeSortedUserArray(id, dataSet) {
```
Comment (nataliemcintyre2021) [inline src/User-repo.js:32]: Continue to research how Date object is necessary here

File: src/User-repo.js [hunk 4]
```diff
@@ -1,30 +1,51 @@
...
+    // var totalStepGoal = this.users.reduce((sumSoFar, data) => {
+    //   return sumSoFar = sumSoFar + data.dailyStepGoal;
+    // }, 0);
+    // return totalStepGoal / this.users.length;
+
+    const totalStepGoal = this.users.reduce((total, user) => {
+      total += user.dailyStepGoal;
+      return total
     }, 0);
+    console.log(totalStepGoal / this.users.length, "nina")
```
Comment (ninabrissey) [inline src/User-repo.js:27]: remove this console.log

**Comment by ninabrissey** [comment]
Looks good. Thank you. Natalie and I will work on Fetch calls.

**Pull request merged**